packages/bls/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@chainsafe/bls",
+  "version": "0.1.6",
+  "description": "Implementation of bls signature verification for ethereum 2.0",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "homepage": "https://github.com/chainsafe/bls-js",
+  "author": "ChainSafe Systems",
+  "license": "Apache-2.0",
```

Comment:
do we have to have that license because of apache milagro?

Reply:
I don't think so, I just left it as is.